web/src/components/organisms/Settings/Members/index.tsx

```diff
@@ -85,7 +68,7 @@ const Members: React.FC = () => {
 
   const members = currentWorkspace?.members;
 
-  const showConfirm = useCallback(
+  const handleMemberDelete = useCallback(
```

Comment:
This callback still is accepting anything as member. Now that we have the Member type we can type it, right?